http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/mocks.scala

```diff
@@ -172,17 +170,19 @@ class MockKubernetesService(podStatus: PodStatus = PodStatus.Running, appRelease
     ev: Ask[IO, TraceId]
   ): IO[List[V1PersistentVolumeClaim]] =
     appRelease.flatTraverse { r =>
-      val nfsPvc = mock[io.kubernetes.client.openapi.models.V1PersistentVolumeClaim]
-      val nfcMetadata = mock[V1ObjectMeta]
-      when(nfcMetadata.getName).thenReturn(s"${r.asString}-galaxy-pvc")
-      when(nfcMetadata.getUid).thenReturn(s"nfs-pvc-id1")
-      when(nfsPvc.getMetadata()).thenReturn(nfcMetadata)
-
-      val cvmfsPvc = mock[io.kubernetes.client.openapi.models.V1PersistentVolumeClaim]
-      val cvmfsMetadata = mock[V1ObjectMeta]
-      when(cvmfsMetadata.getName).thenReturn(s"${r.asString}-cvmfs-alien-cache-pvc")
-      when(cvmfsMetadata.getUid).thenReturn(s"cvmfs-pvc-id1")
-      when(cvmfsPvc.getMetadata()).thenReturn(cvmfsMetadata)
+      val nfcMetadata = new V1ObjectMetaBuilder(false).build()
```

Comment:
have to move away from mock becuz I sometimes get `classload` error running locally